### PR TITLE
feat: added Search and more

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:

#Description: Added Search and more features

## More details:
User can perform the following tasks directly from the extension
- Search on Google
- Open Gmail
- Search on Youtube
- Search on Google maps
- Open Calendar
- Open Google meet
- Open Google Drive"
- Open Google docs 

Screenshots:
ext6

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Features thinking awesome.
Code works well

File: manifest.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "manifest_version": 3,
```
Comment (TejasShekar) [inline manifest.json:2]: Make it version 2 if you are uploading it to firefox add on

File: weather.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ const generateWeatherData = (weatherApi) => {
         temp.innerText = Math.round(data.current.temp) + "°";
         imgURL = `http://openweathermap.org/img/wn/${data.current.weather[0].icon}.png`;
         weatherIcon.src = imgURL;
+        console.log(data);
```
Comment (TejasShekar) [inline weather.js]: Remove console logs

**Review by TejasShekar — APPROVED** [review]
- Nice feature of including g-suite as sidebar.
- UI looks good.

**Pull request merged**